cocos/3d/misc/mesh-codec.ts

```diff
@@ -21,13 +21,13 @@
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  THE SOFTWARE.
 */
-import { CULL_MESHOPT, WASM_SUPPORT_MODE } from 'internal:constants';
+import { CULL_MESHOPT, NATIVE_CODE_BUNDLE_MODE } from 'internal:constants';
```

Comment:
I think WASM_SUPPORT_MODE is more meaningful than NATIVE_CODE_BUNDLE_MODE.

Reply:
I don't think so. 
If using asmjs, it's a normal js code. asmjs is not a wasm mode.